yt/geometry/tests/test_particle_octree.py

```diff
@@ -398,6 +406,70 @@ def test_bitmap_select():
                     assert_array_equal(gf, gf_ans, f"selector {i}, ghost file array")
 
 
+@requires_file("TNGHalo/halo_59.hdf5")
+def test_ewah_write_load(tmp_path):
+    mock_file = tmp_path / "halo_59.hdf5"
+    mock_file.symlink_to(_get_test_data_dir_path() / "TNGHalo" / "halo_59.hdf5")
+
+    ds = load(mock_file)
+    _, c = ds.find_max(("gas", "density"))
+    assert ds.index.pii.mutable_index
+    assert ds.index.pii.order1 == 6
+    assert ds.index.pii.order2_orig == 2
+    assert ds.index.pii.order2 == 4
+    assert not ds.index.pii.loaded
+    c += ds.quan(0.5, "Mpc")
+    sp = ds.sphere(c, (20.0, "kpc"))
+    mass = sp.sum(("gas", "mass"))
+
+    del ds
```

Comment:
I don't see what these calls bring. Garbage collection is *not* triggered by `del`, nor is it necessary for the test to function